src/app/data-planes/views/DataPlaneDetailView.vue

```diff
@@ -504,10 +505,62 @@
             data-testid="dataplane-subscriptions"
           >
             <h2>{{ t('data-planes.routes.item.subscriptions.title') }}</h2>
-
-            <KCard class="mt-4">
-              <SubscriptionList :subscriptions="props.data.dataplaneInsight.subscriptions" />
-            </KCard>
+            <AppCollection
+              :headers="[
+                { ...me.get('headers.instanceId'), label: t('http.api.property.instanceId'), key: 'instanceId' },
+                { ...me.get('headers.version'), label: t('http.api.property.version'), key: 'version' },
+                { ...me.get('headers.connected'), label: t('http.api.property.connected'), key: 'connected' },
+                { ...me.get('headers.disconnected'), label: t('http.api.property.disconnected'), key: 'disconnected' },
+                { ...me.get('headers.responses'), label: t('http.api.property.responses'), key: 'responses' },
+              ]"
+              :is-selected-row="item => item.id === route.params.subscription"
+              :items="props.data.dataplaneInsight.subscriptions.map((_, i, arr) => arr[arr.length - (i + 1)])"
```

Comment:
If I am not mistaken, you are reversing `subscriptions`. What about using `Array.reverse`? 🙂 
```suggestion
              :items="props.data.dataplaneInsight.subscriptions.reverse()"
```

Reply:
`reverse` mutates the original array, which we don't want to do. We probably want `toReversed` but at the time I figured it was a bit early to be using that so went with this. I normally aim for a rough ~2 years of support for browser stuff.

---

FYI: We _might_ eventually order these ourselves (but in the data layer), but I usually keep data in the same order as what the backend gives us and then if we need to re-order for the specific view then do it in the specific view itself.

Reply:
Oh you are right, should be `toReversed` 👍 [caniuse](https://caniuse.com/?search=toReversed) reports >91% support (all users), which is good but probably not enough.

Reply:
Resolving this one